src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -283,6 +290,29 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
                     return containedItems;
                 }
 
+                if (listPartFilter.DisplayText != null)
```

Comment:
A `listPartFilter` lehet `null` is, erre gondolj.

Reply:
A `Status` property-knél továbbra is `NullReferenceException` lesz. Amúgy a folyamatos `?` operatorok helyett egyszerűbb, hogy ha a `QueryContainedItemsAsync` legelején ellenőrzöd, hogy `null`-e, és ha igen, ott létrehozol egy üres példányt.